tests/e2e/ekscluster/furyctl_apply.expect

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/expect -f
+# Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+
+# Increase timeout as needed:
+set timeout 3600
+puts "Current working directory: $env(PWD)"
+if {[llength $argv] == 0} {
+    puts "Usage: furyctl_apply.expect /path/to/furyctl.yaml"
+    exit 1
+}
+
+if {[llength $argv] >= 2} { 
+    set outdir [lindex $argv 1] 
+    puts "Using specified output directory: $outdir" 
+} else { 
+    set outdir $env(PWD) 
```

Comment:
FYI: you can set furyctl flags via env vars, for example the env var `FURYCTL_OUTDIR` can be used to set the `--outdir` flag